test/suite/test_lsm02.py

```diff
@@ -59,22 +59,20 @@ def test_lsm_tombstone(self):
         self.add_key(self.uri, 'k3', v)
         self.verify_key_exists(self.uri, 'k3', v)
 
-    def test_lsm_rename01(self):
+    def test_lsm_create01(self):
         self.session.create(self.uri, 'key_format=S,value_format=S')
         self.add_key(self.uri, 'a', 'a')
-        self.renameUntilSuccess(self.session, self.uri, self.uri + 'renamed')
-        self.verify_key_exists(self.uri + 'renamed', 'a', 'a')
+        self.verify_key_exists(self.uri, 'a', 'a')
 
-    def test_lsm_rename02(self):
+    def test_lsm02_create02(self):
```

Comment:
Again, this function can (I think) also go. It's just two creations, insertions, then assertions that the key exists.

Reply:
[4651212](https://github.com/wiredtiger/wiredtiger/pull/10282/commits/4651212791deac3577df5658fab5aa3c0f6f45c4)